build-template/build.cmd

```diff
@@ -16,11 +16,19 @@ copy %CACHED_NUGET% .nuget\nuget.exe > nul
 
 :restore
 IF EXIST packages\KoreBuild goto run
-.nuget\NuGet.exe install KoreBuild -ExcludeVersion -o packages -nocache -pre
+IF DEFINED BUILDCMD_RELEASE (
+	.nuget\NuGet.exe install KoreBuild -version 0.2.1-%BUILDCMD_RELEASE% -ExcludeVersion -o packages -nocache -pre
```

Comment:
Yeah for me this felt like an immediate fix for beta4. We need to revisit this once we decide the versioning story for KoreBuild